dev-util/vulkan-tools/vulkan-tools-9999.ebuild

```diff
@@ -26,7 +26,7 @@ SLOT="0"
 IUSE="cube wayland +X"
 
 # Cube demo only supports one window system at a time
-REQUIRED_USE="!cube? ( || ( X wayland ) ) cube? ( ^^ ( X wayland ) )"
+REQUIRED_USE="cube? ( ^^ ( X wayland ) )"
 
```

Comment:
I think VULKAN_HEADERS_INSTALL_DIR and GLSLANG_INSTALL_DIR need to be ESYSROOT here as per other discussions (and consistency with the others)?

Reply:
Indeed, fixed. I also noticed that `vulkan-loader` was previously fixed in commit fce28df7e8304ff4b93b1c194a1f8cb76876b571.